frontend/src/features/projects/dispose/components/ReviewApproveActions.tsx

```diff
@@ -11,36 +11,37 @@ const FlexRight = styled.div`
   margin: 0.5rem 0;
 `;
 
-interface IReviewApproveActionsProps {
-  //onSave: () => void; TODO
-  //onNext: () => void; TODO
-}
-
 /**
  * A component for project review actions
  * @component
  */
-export const ReviewApproveActions: React.FC<IReviewApproveActionsProps> = () => {
+export const ReviewApproveActions: React.FC = () => {
   const { values, submitForm } = useFormikContext<any>();
   return (
     <>
       <FlexRight>Approve for Enhanced Referral Process</FlexRight>
       <FlexRight>
         <span>
           <Button
+            disabled={
```

Comment:
Since there is no transition after this form is submitted, some of the buttons must be disabled based on the project status.